internal/boxcli/cache.go

```diff
@@ -64,7 +65,7 @@ func cacheCmd() *cobra.Command {
 	cacheCommand.AddCommand(uploadCommand)
 	cacheCommand.AddCommand(cacheConfigureCmd())
 	cacheCommand.AddCommand(cacheCredentialsCmd())
-	cacheCommand.Hidden = true
```

Comment:
Should we add a caveat to the command help saying it's only in private beta for now?

Reply:
The upload command works without Jetify cloud (you can specify your own s3 bucket and pass in your own credentials) so I thought it would be cool to open it up. The credentials and config commands are still hidden.